app/src/main/java/ch/epfl/sdp/drone3d/service/impl/mission/ParallelogramMappingMissionService.kt

```diff
@@ -15,8 +16,13 @@ import javax.inject.Inject
 class ParallelogramMappingMissionService @Inject constructor(val droneService: DroneService): MappingMissionService {
 
     private val cameraAngle = 0.0 // Suppose the drone is looking down
+    //Those default data correspond to the Freefly Astro Quadrotor properties
+    private val defaultSensorWidth = 7.82f
+    private val defaultFocalLength = 24.0f
+    private val defaultImageWidth = 1280
+    private val defaultImageHeight = 720
```

Comment:
Would be nice to create a companion object

Reply:
Yes putting everything in a companion object would be better since these variables are common to every instances of the class. Also adding const would be better

Reply:
You are right, I have corrected it